Sources/BraintreePaymentFlow/BTPaymentFlowClient.m

```diff
@@ -39,7 +39,7 @@
 #import <BraintreeCoreSwift/BraintreeCoreSwift-Swift.h>
 #endif
 
-@interface BTPaymentFlowClient () <SFSafariViewControllerDelegate>
+@interface BTPaymentFlowClient () <SFSafariViewControllerDelegate, BTAppContextSwitchClient>
```

Comment:
Moved this protocol conformance from the header file to the `.m` to make Cocoapods happy. Cocoapods won't let you import an outside framework in a `.h` file